### ISSUE TITLE
TypeError: Cannot read property 'register' of undefined

### ISSUE THREAD

**Issue by javidjamae** [expo/expo#412]
If I try to use Jest to test any component with a Modal in it and I have an import anywhere in the code that references Permissions (from Expo) I get the following error:

```
React caught an error thrown by Modal. You should fix this error in your code. Consider adding an error boundary to your tree to customize error handling behavior.

TypeError: Cannot read property 'register' of undefined

The error is located at:
    in Modal

The error was thrown at:
    at Modal.componentWillMount ([PROJECT_DIR]/node_modules/expo/src/Modal/Modal.js:15:42)
```

This is the minimal Jest test necessary to reproduce it:
```
import 'react-native'
import React from 'react'

import { Modal } from 'react-native'
import { Permissions } from 'expo'

import renderer from 'react-test-renderer'

test( 'renders correctly', () => {
  const tree = renderer.create(
    <Modal />
  ).toJSON()

  expect( tree ).toMatchSnapshot()
} )
```

If I comment out the line where it imports Permissions from Expo, the test runs properly.

**Comment by brentvatne** [comment]
hi there! I know what is going on here and will fix it tomorrow :)

**Comment by brentvatne** [comment]
had a chance to fix it today, install `jest-expo@19.0.1` and it should work as expected

**Comment by javidjamae** [comment]
@brentvatne that was fast, thanks!
